pyproject.toml

```diff
@@ -13,7 +13,21 @@ aiohttp = "^3.8.3"
 [tool.poetry.dev-dependencies]
 pytest = "^5.2"
 python-dotenv = "^0.21.0"
+pre-commit = "^2.20.0"
+black = "^22.8.0"
+isort = "^5.10.1"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.black]
+line-length = 99
+
+[tool.isort]
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+use_parentheses = true
+ensure_newline_before_comments = true
+profile=black
```

Comment:
GitHub is highlighting this line - Is there a problem / invisible character?

Reply:
I guess not 
There shouldn't be any invisible character or a problem here
I typed it in so there shouldn't be any problem